src/test/java/no/sikt/nva/scrapers/EntityDescriptionExtractorTest.java

```diff
@@ -34,4 +40,18 @@ var record = dublinCoreScraper.validateAndParseDublinCore(dublinCore,
         var descriptions = record.getEntityDescription().getDescriptions();
         assertThat(descriptions, hasItem(someLocalCode));
     }
+
+    @Test
+    void shouldKeepOriginalOrderOfDescriptions() {
+        var dublinCore = DublinCoreFactory.createDublinCoreFromXml(new File(TEST_RESOURCE_PATH +
+                                                                            "dc_with_duplicated_values.xml"));
+        var dublinCoreScraper = new DublinCoreScraper(ONLINE_VALIDATION_DISABLED,
+                                                      LOOKUP_IN_CHANNEL_REGISTER,
+                                                      Map.of());
+        var record = dublinCoreScraper.validateAndParseDublinCore(dublinCore,
+                                                                  new BrageLocation(null),
+                                                                  SOME_CUSTOMER);
+        var descriptions = record.getEntityDescription().getDescriptions();
+        assertThat(descriptions, is(equalTo(Set.of("First description", "Second description"))));
```

Comment:
There is no order to a set according to Set definitions.

Reversing the order of the elements would still make the test pass.